internal/agent/device/fileio/writer.go

```diff
@@ -66,6 +66,51 @@ func (w *writer) RemoveFile(file string) error {
 	return nil
 }
 
+func (w *writer) CopyFile(src, dst string) error {
+	return w.copyFile(filepath.Join(w.rootDir, src), filepath.Join(w.rootDir, dst))
+}
+
+func (w *writer) copyFile(src, dst string) error {
```

Comment:
There is small project github.com/otiai10/copy for copying files, maybe we can use it

Reply:
we want to reduce deps on the agent but we can consider it.